pkg/handler/eventhandler_test.go

```diff
@@ -888,7 +998,14 @@ var _ = Describe("Eventhandler", func() {
 						IsInInitialList: false,
 					}, wq)
 
-					Expect(actualOpts).To(Equal(priorityqueue.AddOpts{}))
+					if test.overridePriority != 0 {
+						Expect(actualOpts).To(Equal(priorityqueue.AddOpts{After: test.after, RateLimited: test.ratelimited, Priority: ptr.To(test.overridePriority)}))
+					} else {
+						Expect(actualOpts).To(Or(
+							Equal(priorityqueue.AddOpts{After: test.after, RateLimited: test.ratelimited}),
+							Equal(priorityqueue.AddOpts{After: test.after, RateLimited: test.ratelimited, Priority: ptr.To(0)}),
```

Comment:
Wondering if this line should be here (if we make the changes I suggested in eventhandler.go)

(same in l.1081)